dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/aroundconstruct/AroundConstructTestServlet.java

```diff
@@ -100,6 +136,11 @@ public void testInterceptorOrder() {
 
     }
 
+    /**
+     * Interceptors should only be called once for each constructor
+     */
+    @Test
+    @Ignore //TODO need to find out why this test does not pass
     public void testInterceptorNotCalledTwice() {
```

Comment:
Is there an issue for this?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/15711